common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -46,6 +46,16 @@ trait ABTestSwitches {
     exposeClientSide = true
   )
 
+  Switch(
+    ABTests,
+    "ab-acquisitions-banner-sign-in-cta",
+    "test whether having a sign-in CTA negatively impacts acquisitions",
+    owners = Seq(Owner.withGithub("Mullefa")),
```

Comment:
Up to you but you can set the owner to Owner.group(SwitchGroup.Identity), we have done that in the past with Identity switches